Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Search/OneWay.hs

```diff
@@ -91,7 +141,12 @@ oneWaySearch ::
 oneWaySearch personId req bundleVersion clientVersion device = do
   person <- QP.findById personId >>= fromMaybeM (PersonDoesNotExist personId.getId)
   merchant <- QMerc.findById person.merchantId >>= fromMaybeM (MerchantNotFound person.merchantId.getId)
+  mbFavourite <- CSavedLocation.findByLatLonAndRiderId personId req.origin.gps
```

Comment:
i remember discussing this can be sent from frontend and only if its not sent we will do this query.

Reply:
it was discussed for special zone not for favourite

Reply:
shall I take favourite too?

Reply:
I cannot do for favourite as I need is moved data